locales/pt.json

```diff
@@ -89,127 +89,127 @@
+ 	"Tried to load testrpc but an error occurred. This is a problem with testrpc": "Tentativa de carregar testrpc falhou devido a um erro. Isto é um problema com o testrpc",
+ 	"IMPORTANT: if you using a NodeJS version older than 6.9.1 then you need to install an older version of testrpc \"%s\". Alternatively install node 6.9.1 and the testrpc 3.0": "IMPORTANTE: se vocês está utilizando um versão do NodeJS menor que 6.9.1, você precisará instalar uma versão antiga do testrpc \"%s\". Como alternativa, você pode instalar o Node 6.9.1 e o testrpc 3.0",
+ 	"terminating due to error": "finalizando por causa de erro",
+ 	"There a a space in the version of {{versionKey}}. We corrected it for you ({{correction})": "Existe um espaço na versão {{versionKey}}. Carrigimos para você ({{correction})",
+ 	"versions": "versões",
+ 	"versions in use": "versões em uso",
+ 	"downloading {{packageName}} {{version}}....": "baixando {{packageName}} {{version}}....",
+ 	"language to use (default: en)": "linguagem a ser utilizada (padrão: en)",
+ 	"download template": "download concluído",
+ 	"Start the Embark console": "Iniciar o console Embark",
+ 	"When set, will print the gas cost for each contract deploy": "Quando configurado, irá mostrar o cussto do gás para cada contrato publicado",
+ 	"Graph will not include undeployed contracts": "Grafo não irá incluir contratos não publicados",
+ 	"Graph will not include functions": "Grafo não irá incluir funções",
+ 	"Graph will not include events": "Grafo não irá incluír eventos",
+ 	"ENS domain to associate to": "Domínio ENS para se associar com",
+ 	"output usage information and help information": "informação de saída de uso e informação de ajuda",
+ 	"Starting %s process": "Iniciando processo %s",
+ 	"Starting Blockchain node in another process": "Iniciando Nó de Blockchain em outro processo",
+ 	"Blockchain node is ready": "Nó de Blockchain está pronto",
+ 	"Storage process for {{storageName}} ended before the end of this process. Code: {{code}}": "Processo de armazenamento para {{storageName}} finalizou antes que pudesse acabar de processar. Código: {{code}}",
+ 	"ipfs process started": "processo ipfs iniciado",
+ 	"Compiling...": "Compilando...",
+ 	"Building...": "Construindo...",
+ 	"gas at the price of": "gás no preço de",
+ 	"Wei, estimated cost:": "Wei, custo estimado:",
+ 	"using": "usando",
+ 	"Embark is building, please wait...": "Embark está em fase de construção, por favor, aguarde...",
+ 	"instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)": "objeto js-ipfs instanciado configurado para o ambiente atual (disponível se ipfs está ativado)",
+ 	"Initializing Embark Template...": "Iniciando Template do Embark...",
+ 	"Name your app (default is %s):": "Nome do aplicativo (o padrão é %s):",
+ 	"Unrecognized balance string \"%s\"": "Texto de balanceamento desconhecida: \"%s\"",
```

Comment:
`balanceamento` here is wrong. This is used in the context of an account balance.

Reply:
WDYT: ```Texto de saldo...```?